pkg/state/state_ofed.go

```diff
@@ -36,8 +37,13 @@ import (
 	"github.com/Mellanox/network-operator/pkg/utils"
 )
 
-const stateOFEDName = "state-OFED"
-const stateOFEDDescription = "OFED driver deployed in the cluster"
+const (
+	stateOFEDName        = "state-OFED"
+	stateOFEDDescription = "OFED driver deployed in the cluster"
+
+	// TODO: update with new version
+	newMofedImageFormatVersion = "5.7-0.0.8.1"
```

Comment:
What about version with two digits, e.g. `5.10-0.0.8.1`?

Reply:
good catch !  i will use https://github.com/Masterminds/semver

Reply:
PTAL, also added UT to cover this